docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
+
+.. figure::  images/filtered_unweighted_beta.png
+   :align:   center
+
+Several parameters have been automatically selected for you. In the "p-metric" field enter "unweighted Unifrac" and in the "i-tree" field enter "/databases/gg/13_8/trees/97_otus.tree", and click "Add Command". 
+
+To create an unweighted beta analysis principal coordinate plot, select select "pcoa" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/filtered_unweighted_pcoa.png
+   :align:   center
+
+All of the parameters have been automatically selected for you just click "Add Command". Once the command is added the workflow should appear as follows:
+
+.. figure::  images/filtered_unweighted_workflow.png
+   :align:   center
+
+Click the run button to start the process of the beta diversity analysis. The view will return to the original screen, while the beta diversity analysis job runs.
+
+Clicking on the pcoa (Principal Coordinate Analysis) results will open an interactive visualization of the similarity among your samples. Generally speaking, the more similar the samples, the closer the are likely to be in the PCoA ordination. The Emperor visualization program offers a very useful way to explore how patterns of similarity in your data associate with different metadata categories.
+
+Once the Emperor visualization program loads, the PCoA result will look like: 
+
+.. figure::  images/full_pcoa.png
+   :align:   center
+
+You will see tabs including "Color", "Visibility", "Shape", "Axes", and "Scale"
+
+Under "Color" you will notice two pull-down menus:
+
+.. figure::  images/color_tab.png
+   :align:   center
+ 
+Under "Select a Color Category" you can select how the samples will be grouped. Under "Classic QIIME Colors", you can select how each group will be colored.
+
+Under the "Visibility" tab you will notice 1 pull-down menu:
+
+.. figure::  images/visibility_tab.png
+   :align:   center
+ 
+Under "Select a Visibility Category" you can select which group will be displayed on the PCoA plot.
+
+Under the "Shape" tab you will notice 1 pull-down menu:
+
+.. figure::  images/shape_tab.png
+   :align:   center
+
+Under "Select a Shape Category" you can alter the shape of each group on the PCoA plot to the following:
+
+.. figure::  images/shape_options.png
+   :align:   center
+
+Under the "Axis" tab you will notice 5 pull-down menus:
+
+.. figure::  images/axis_tab.png
+   :align:   center
+
+The first 3 pull-down menus located under "Visible" allow you to change the axis that are being displayed.
+The "Axis and Labels Color" pull-down menu allow you to change the color of your axis and label of the PCoA.
+The "Background Color" pull-down menu allows you to change the color of the background of the PCoA.
+The % Variantion Expanded graph displays how different the most dissimilar samples are by percentage for each axis that can be used. 
```

Comment:
too many "pull-down" menus. I think it's fine to say, e.g. "Background Color" menu.